tests/data/aea-config.example_multipage.yaml

```diff
@@ -0,0 +1,74 @@
+agent_name: myagent
+author: fetchai
+version: 0.2.0
+description: An example of agent configuration file for testing purposes.
+license: Apache-2.0
+aea_version: '>=0.5.0, <0.6.0'
+fingerprint: {}
+fingerprint_ignore_patterns: []
+connections:
+- fetchai/oef:0.7.0
+contracts: []
+protocols:
+- fetchai/oef_search:0.4.0
+- fetchai/default:0.4.0
+- fetchai/tac:0.5.0
+- fetchai/fipa:0.5.0
+skills:
+- fetchai/echo:0.5.0
+default_connection: fetchai/oef:0.7.0
+default_ledger: cosmos
+logging_config:
+  disable_existing_loggers: false
+  version: 1
+private_key_paths:
+   cosmos: cosmos_private_key.txt
+   ethereum: eth_private_key.txt
+connection_private_key_paths:
+   cosmos: cosmos_private_key.txt
+   ethereum: eth_private_key.txt
+registry_path: ../../packages
+---
+name: dummy
+author: dummy_author
+version: 0.1.0
+type: skill
```

Comment:
Note; this should by definition have to be also listed as a dependency above under skills

Reply:
right!

Reply:
I think this should include also a connection spec; so we test overriding that too